clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -874,9 +874,13 @@ public void subscribe(Pattern pattern, ConsumerRebalanceListener listener) {
     public void unsubscribe() {
         acquire();
         try {
+            //TODO: Reactivate the commented code once KAFKA-3491 is resolved
```

Comment:
Sure. I was planning on opening another JIRA to re-enable those sections.
By first part, do you mean removing the line above (#877)?